app/libs/installations/slack/api.rb

```diff
@@ -68,9 +69,16 @@ def list_channels(transforms, cursor = nil)
       execute(:get, LIST_CHANNELS_URL, params)
         .then { |response| response.slice("channels", "response_metadata") }
         .then { |responses|
-        {channels: Installations::Response::Keys.transform(responses["channels"], transforms),
-         next_cursor: responses.dig("response_metadata", "next_cursor")}
-      }
+          {channels: Installations::Response::Keys.transform(responses["channels"], transforms),
+           next_cursor: responses.dig("response_metadata", "next_cursor")}
+        }
+    end
+
+    def team_info(transforms)
+      execute(:get, GET_TEAM_URL, {})
+        .then { |response| response["team"] }
```

Comment:
Safe traverse?

Reply:
Instead of safe traversal, I should add handling 4xx and 5xx from Slack. Because success response will always have `team` key.